### PR TITLE
#1-4Task. Julia.D

### PR DESCRIPTION
**Pull request by JuliaDer**
https://travis-ci.org/github/JuliaDer/js-assignments/builds/703996440

### CODE REVIEW

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -127,7 +137,7 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
+    return value % 10;
 }
```
Comment (IlyaMokin) [inline task/02-numbers-tasks.js:141]: !

File: task/03-date-tasks.js [hunk 2]
```diff
@@ -76,7 +77,13 @@ function isLeapYear(date) {
...
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+   const diff = endDate - startDate
+   const date = new Date(endDate - startDate)
+   const hours = ('00' + Math.floor(diff / (3600 * 1000))).slice(-2)
+   const minutes = ('00' + date.getMinutes()).slice(-2)
+   const seconds = ('00' + date.getSeconds()).slice(-2)
+   const milliSeconds = ('000' + date.getMilliseconds()).slice(-3)
+   return `${hours}:${minutes}:${seconds}.${milliSeconds}`
```
Comment (IlyaMokin) [inline task/03-date-tasks.js:86]: cool

**Comment by IlyaMokin** [comment]
Looks quite cool! Closing the PR as accepted!
Thanks!

File: task/03-date-tasks.js (hunk 2 above)
Comment (JuliaDer) [reply task/03-date-tasks.js:86]: Thank you! :)

**Pull request closed without merging**